Sources/Core/Selection/FilterSelectionStore.swift

```diff
@@ -66,6 +82,8 @@ extension FilterSelectionStore {
         } else {
             setValue(filter.value, for: filter)
         }
+
+        delegate?.filterSelectionStoreDidChange(self)
```

Comment:
Is it ok that this delegate method will be called at least twice: once in either `removeValues` or `setValue` and then here in `toggleValue`? Also when we remove value for filter it will iterate through sub-filters and make delegate calls for each of them.

Reply:
I think we should try to avoid this. Maybe we could make the recursive methods private?